docs/source/pandas_df_conversion.rst

```diff
@@ -0,0 +1,252 @@
+
+        **pandas.DataFrame** 
+        
+        consensus map meta data and intensity stored in pandas DataFrame
+
+**pyopenms.ConsensusMap.get_intensity_df()**
+        Generates a pandas DataFrame with feature intensities from each sample in long format (over files).
+        For labelled analyses channel intensities will be in one row, therefore resulting in a semi-long/block format.
+        Resulting DataFrame can be joined with result from get_metadata_df by their index 'id'.
+
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        intensity DataFrame
+
+**pyopenms.ConsensusMap.get_metadata_df()**
+        Generates a pandas DataFrame with feature meta data (sequence, charge, mz, RT, quality).
+        Resulting DataFrame can be joined with result from get_intensity_df by their index 'id'.
+
+        **Returns:**
+
+        **pandas.DataFrame** 
+        
+        DataFrame with metadata for each feature (such as: best identified sequence, charge, centroid RT/mz, fitting quality)
+
+**Examples:**
+
+.. code-block:: python
+
+    urlretrieve(url+'ConsensusXMLFile_1.consensusXML', 'ConsensusXMLFile_1.consensusXML')
+    consensus_map = ConsensusMap()
+    ConsensusXMLFile().load('ConsensusXMLFile_1.consensusXML', consensus_map)
+
+    df = consensus_map.get_df()
+    df.head(2)
+    
+.. csv-table:: consensus_map.get_df()
+   :widths: 2 10 20 20 20 20 30 30
+   :header:     , "sequence",	"charge",	"RT",	"mz",	"quality",	"data/MapAlignmentFeatureMap1.xml",	"data/MapAlignmentFeatureMap2.xml"
+            "id"
+
+   "0",	"A",	"0",	"1273.27",	"904.470",	"1.1",	"31253900.0",	"0.0"
+   "1",	"E",	"0",	"1248.33",	"897.449",	"1.2",	"25917900.0",	"25917900.0"
+
+.. code-block:: python
+
+    df = consensus_map.get_intensity_df()
+    df.head(2)
+
+.. csv-table:: consensus_map.get_intensity_df()
+   :widths: 2 30 30
+   :header:     , "data/MapAlignmentFeatureMap1.xml",	"data/MapAlignmentFeatureMap2.xml"
+            "id"
+
+   "0",	"31253900.0",	"0.0"
+   "1",	"25917900.0",	"25917900.0"
+
+.. code-block:: python
+
+    df = consensus_map.get_metadata_df()
+    df.head(2)
+
+.. csv-table:: consensus_map.get_metadata_df()
+   :widths: 2 10 20 20 20 20
+   :header:     , "sequence",	"charge",	"RT",	"mz",	"quality"
+            "id"
+
+   "0",	"A",	"0",	"1273.27",	"904.470",	"1.1"
+   "1",	"E",	"0",	"1248.33",	"897.449",	"1.2"
+
+PeptideIdentifications
+**********************
+
+**pyopenms.peptide_identifications_to_df( peps**, *decode_ontology=True*, *default_missing_values={bool: False, int: -9999, float: np.nan, str: ''}*, *export_unidentified=True* **)**
+        Generates a pandas DataFrame with all peaks in the MSExperiment
+
+        **Parameters:**
+
+        **peps :** 
+        
+        list of PeptideIdentification objects
+
+        **decode_ontology :** default True
+        
+        decode meta value names
+
+        **default_missing_values :** default {bool: False, int: -9999, float: np.nan, str: ''}
```

Comment:
is there a better way than using -9999 ?

Reply:
In numpy np.nan is only for floats, there is no missing value for ints. This is just a fallback, the user can also change.

Reply:
I see. Ok then let's keep the sentinel